templates/dmarc_reports/index.html.twig

```diff
@@ -8,74 +8,87 @@
+                                  <td class="text-nowrap">
+                                    {% set dkim_pass = true %}
+                                    {% set spf_pass = true %}
+                                    {% for record in report.dmarc_records %}
+                                        {% if record.policydkim != "pass" %}
+                                          {% set dkim_pass = false %}
+                                        {% endif %}
+                                        {% if record.policyspf != "pass" %}
+                                          {% set spf_pass = false %}
+                                        {% endif %}
+                                    {% endfor %}
+                                    <span class="badge badge-pill text-bg-{% if spf_pass %}success{% else %}danger{% endif %}">SPF</span>
+                                    <span class="badge badge-pill text-bg-{% if dkim_pass %}success{% else %}danger{% endif %}">DKIM</span>
+                                  </td>
+                                  <td class="text-nowrap">
+                                    <a href="{{ path('app_dmarc_reports_report', {report: report.id}) }}" class="link-primary px-2" title="View details">
+                                      <!-- Bootstrap icon 'eye-fill' -->
+                                      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-eye-fill" viewBox="0 0 16 16">
+                                        <path d="M10.5 8a2.5 2.5 0 1 1-5 0 2.5 2.5 0 0 1 5 0"/>
+                                        <path d="M0 8s3-5.5 8-5.5S16 8 16 8s-3 5.5-8 5.5S0 8 0 8m8 3.5a3.5 3.5 0 1 0 0-7 3.5 3.5 0 0 0 0 7"/>
+                                      </svg></a>
+                                    <a href="{{ path('app_dmarc_reports_delete', {report: report.id}) }}" class="link-danger" title="Delete entry ..." onclick="return confirm('{{'Are you sure? this can not be undone'|trans}}');">
+                                      <!-- Bootstrap icon 'trash-fill' -->
+                                      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-trash-fill" viewBox="0 0 16 16">
+                                        <path d="M2.5 1a1 1 0 0 0-1 1v1a1 1 0 0 0 1 1H3v9a2 2 0 0 0 2 2h6a2 2 0 0 0 2-2V4h.5a1 1 0 0 0 1-1V2a1 1 0 0 0-1-1H10a1 1 0 0 0-1-1H7a1 1 0 0 0-1 1zm3 4a.5.5 0 0 1 .5.5v7a.5.5 0 0 1-1 0v-7a.5.5 0 0 1 .5-.5M8 5a.5.5 0 0 1 .5.5v7a.5.5 0 0 1-1 0v-7A.5.5 0 0 1 8 5m3 .5v7a.5.5 0 0 1-1 0v-7a.5.5 0 0 1 1 0"/>
+                                      </svg></a>
+                                  </td>
+                                </tr>
+                              {% endfor %}
+                              </tbody>
+                            </table>
+                      </div>
+                      <div>
+                        <span class="d-flex justify-content-center">
```

Comment:
i had it on the right for the simple reason that its always at the same spot, even with pagination numbers added.

Reply:
yeah they should be more outstanding but it wasnt a priority so far :)
it will be updated

Reply:
How about top right? Easier accessibility, saves screen space, fixed position, and room for a page number dropdown if so later desired.
![image](https://github.com/antedebaas/DMARC-SMTPTLS-Reports/assets/44540/6b721562-cf3b-4204-822e-60b90feaf4b2)

Reply:
I didn't commit this, so fine by me.